doc/quickstart/geoserver_quickstart.rst

```diff
@@ -267,31 +232,23 @@ Now click :guilabel:`Save` and go to the Layer Preview page to check that it loo
     can see the new image. 
 
 
-Clients for WMS layers
-================================================================================
-
-A large variety of clients exist to make use of the `WMS <http://www.opengeospatial.org/standards/wms>`__ layers you are serving
-from |GS|. This is a list of just some of them 
-
-    * :doc:`uDig <../overview/udig_overview>`
-
-    * :doc:`OpenLayers <../overview/openlayers_overview>`
-
-    * :doc:`MapBender <../overview/mapbender_overview>`
```

Comment:
Any reason why this section has been removed. From my perspective it cross-reference other packages distrubuted by OSGeo live and its worth for users to see how these might work togeher

Reply:
Okay, I will re-instate. It seemed out of place but now that you've explained it, it makes sense.